project_2.html

```diff
@@ -0,0 +1,603 @@
+			<p class="mb-5">
+				<img class="img-fluid my-3 me-5 float-end" style="max-width: 300px;"
+					src="./assets/images/visuals/ops_piezo.webp">
+				A <strong>piezoelectric buzzer</strong> is a component used to emit sound. 
+				<br><br>
+				Piezo buzzers come in two varieties: active and passive. 
+				<strong>Active</strong> buzzers have a built-in oscillator with a set frequency that cannot be changed. 
+				<strong>Passive</strong> buzzers do not; they have to be fed an oscillating signal.
+				<br><br>
+				We will use a passive piezo buzzer.
+			</p>
+			
+			<h4 class="mb-3 fw-light" id="pj-buzzer_symbol" style="display: inline;">Symbol</h4>
+			<p class="mb-5">
+				<img class="img-fluid my-3 me-5 float-end" style="max-width: 120px;"
+					src="./assets/images/visuals/ops_symbol_piezo.png">
+				The symbol on the right depicts a non-polar piezo buzzer. The symbols for speakers and piezo buzzers 
+				are often used interchangeably.
+			</p>
+
+			<!--Finished up to here-->
+
+			<h3 class="mb-3" id="pj-heading-NE555P_IC">NE555P IC</h3>
+			<p class="mb-3">
+				The <strong>555 Timer</strong> is one of the most popular integrated circuits for hobby projects. It has multiple use cases, 
+				but we will operate the timer in <strong>astable mode</strong>. 
+				In this mode, it outputs an <strong>oscillating digital waveform</strong> (see the image below), 
+				which we will feed to the piezo buzzer.
+				<br>
+				<img class="img-fluid my-3 me-5 float-start" style="max-width: 500px;"
+					src="./assets/images/visuals/ops_waveform_digital.png">
+			</p>
+
+			<p class="mb-5">
+			Remember that the frequency of the waveform is the same as the pitch of the buzzer sound.
+			<br><br>
+		    </p>
+
+			<h4 class="mb-3 fw-light" id="pj-heading-NE555PIC" style="display: inline;">Pinout Diagram</h4>	
+			<p class="mb-3">
+				We will use the <strong>NE555P</strong>, a variant of the 555 Timer. Its pinout diagram is shown below. 
+				<img class="img-fluid my-3 me-5 float-start" style="max-width: 400px;"
+					src="assets/images/visuals/ops_pinout_ne555p.png">
+			</p>
+
+			<p class="mb-5">
+					The <strong>pinout diagram</strong> identifies each contact or pin of the IC. Usually, a datasheet will 
+					include a pinout and a table that matches each pin with a description. The pinout may 
+					change depending on the IC’s packaging, which the datasheet will indicate. For this 
+					project, the IC package is DIP.
+					<br>
+					Note that the organization of the pinout differs from the schematics below. The true location of the ICs pins is shown by the 
+					pinout, not the schematic. Use the notch on a DIP-style IC to find its correct orientation.	
+					<br><br>	
+			</p>
+
+			<h4 class="mb-3 fw-light" id="pj-heading-NE555PIC" style="display: inline;">How It Works</h4>	
+			<p class="mb-5">
+				Learn more about 555 Timer’s functionality in the lecture material <span class="text-blue">here [Webpage Link]</span>.

```

Comment:
![image](https://github.com/uci-ieee/ops-webpages-2023-2024/assets/45702229/6367b580-cda2-481c-990f-bafbf1023e59)

Please replace this placeholder with a link to the lectures page.

We could rewrite this line to say "Learn more about the 555 Timer's functionality in the Lecture II material here"